### PR TITLE
feat: make ProfNITT journey image responsive

### PR DESCRIPTION
**Pull request by Garvit-tyagi**
mobile reponsive images done

### CODE REVIEW

**Comment by 9401adarsh** [comment]
Good to go ahead

File: assets/css/style.css [hunk 1]
```diff
@@ -993,8 +993,25 @@ section {
 --------------------------------------------------------------*/
 .pricing {
   background-color: #f9f9f9;
+
```
Comment (sidvenu) [inline assets/css/style.css]: Not necessary

File: assets/css/style.css [hunk 2]
```diff
@@ -993,8 +993,25 @@ section {
...
+.pricing img {
+  
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: auto;
+ /* object-fit: cover;
+width: 320px;
+  height: auto; */
```
Comment (sidvenu) [inline assets/css/style.css]: Not necessary

File: index.html [hunk 3]
```diff
@@ -335,7 +335,10 @@ <h4>Team Member 4</h4>
 
         <div class="section-title">
           <h2>Journey</h2>
-          <img src="journey.jpeg">
+
+          <div class="wrapper">
+          <img src="journey.jpeg" class="pricing">
+        </div>
```
Comment (sidvenu) [inline index.html]: Format this properly

**Review by sidvenu — CHANGES_REQUESTED** [review]

**Comment by shyam-king** [comment]
touching the "events" arrow in the navbar (mobile) itself scrolls to the events section @Garvit-tyagi

**Comment by shyam-king** [comment]
> touching the "events" arrow in the navbar (mobile) itself scrolls to the events section @Garvit-tyagi

This one @Garvit-tyagi let;s do in another PR. I am merging this PR for now.

**Pull request merged**